app/assets/stylesheets/leaderboard.css

```diff
@@ -58,32 +67,74 @@
 }
 
 .platform-column{
-  width: calc(150 * var(--fpx));
+  width: calc(96 * var(--fpx));
 }
 
 .location-column{
+  width: calc(96 * var(--fpx));
+}
+
+.leaderboard-item.location-column{
+  padding-left: calc(24 * var(--fpx));
+}
+
+.leaderboard-item.platform-column::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: calc(18 * var(--fpx));
+  background-image: var(--ico-url);
+  background-size: auto 100%;
+  background-position: left center;
+  background-repeat: no-repeat;
+  transition: background-image 360ms ease-out;
+}
+
+.leaderboard-item-wrapper:hover .platform-column::before {
+  background-image: var(--ico-url-hover);
+  transition: background-image 72ms ease-out;
+}
+
+.location-flag {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: calc(18 * var(--fpx));
+  height: calc(18 * var(--fpx));
+  border-radius: 50%;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
I'm wondering if we could just show the flags in the location column, without the country code, then make them a little bigger?

Then center all of the contents of the data columns, just leaving the group name left-aligned

My thoughts are 
- this would make the flag easier to see
- centering the data would allow for centering the platform icons while keeping the layout consistent between columns

Reply:
Ahh I totally forgot that I was writing the above comment.